extensions/wikia/AdEngine/js/video/uapVideo.js

```diff
@@ -2,11 +2,17 @@
 define('ext.wikia.adEngine.video.uapVideo', [
 	'ext.wikia.adEngine.adHelper',
 	'ext.wikia.adEngine.context.uapContext',
+	'ext.wikia.adEngine.template.porvata',
+	'ext.wikia.adEngine.template.playwire',
+	'ext.wikia.adEngine.video.player.ui.closeButtonFactory',
+	'ext.wikia.adEngine.video.player.ui.pauseOverlayFactory',
+	'ext.wikia.adEngine.video.player.ui.progressBarFactory',
+	'ext.wikia.adEngine.video.player.ui.volumeControlFactory',
 	'ext.wikia.adEngine.video.uapVideoAnimation',
-	'ext.wikia.adEngine.video.videoAdFactory',
+	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (adHelper, uapContext, uapVideoAnimation, videoAdFactory, log, win) {
+], function (adHelper, uapContext, porvata, playwire, closeButtonFactory, pauseOverlayFactory, progressBarFactory, volumeControlFactory, uapVideoAnimation, doc, log, win) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 12 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)